pdcupdater/utils.py

```diff
@@ -475,6 +481,18 @@ def interesting_tags():
     return stable_tags + [rawhide_tag()]
 
 
+def interesting_container_tags():
+    """ Returns a list of "interesting tags" relevant to containers.
+
+    Eventually, we should query PDC itself to figure out what tags we should be
+    concerned with.
+    """
+    return ['f24-docker']  # This is all we're producing for now...
+    #tags = interesting_tags()
```

Comment:
In general I like to remove commented out code from a repository, but I'll leave it up to you if you want to keep it there since it's only a few lines.